Update paper-advanced.md

BrutusBondBTC opened the pull request:
Turned off Desktop Audio to remove buzzing noise in the Tails video and added how to make an air-gapped computer.

BrutusBondBTC committed:
Update paper-advanced.md

Turned off Desktop Audio to remove buzzing noise in the Tails video and added how to make an air-gapped computer.

BrutusBondBTC commented:
Do I need to run the python program to run the spell check?

mflaxman commented on _pages/setup-wallets/paper-advanced.md:
```diff
@@ -19,8 +20,18 @@ Follow [the official Tails installation instructions](https://tails.boum.org/ins
+
+**Watch A Video On How To Remove The Hard Drive and Wifi Card**
+
+[![AirGapComp](https://nostr.build/i/nostr.build_228a690dc06893b18976af8242284169329274d544ff0e1d22ea79015b338ee0.png)](https://youtu.be/AQuSs2LdARo)
```
Is this anchor text intentional? It doesn't really make sense.

mflaxman commented on _pages/setup-wallets/paper-advanced.md:
```diff
@@ -19,8 +20,18 @@ Follow [the official Tails installation instructions](https://tails.boum.org/ins
 If you are truly paranoid, remove the hard drive from the machine (before booting up Tails) so there is no possibility that any private key material can be written to disk.
 
+
+**Watch A Video On How To Remove The Hard Drive and Wifi Card**
```
This is spelled `Wi-Fi`

mflaxman reviewed (changes requested):
Hotlinking is a bad practice. Please upload static assets to https://github.com/btcguide/btcguide.github.io/tree/master/assets/img

Thank you for the PR @BrutusBondBTC!

BrutusBondBTC committed:
Update paper-advanced.md

Fixed the hotlinking.

I also just eliminated the pictures. The anchor was intentional because I thought it looked better, but there is really no reason to have the pictures for the video anyway. I also think it might be a little confusing to include them after all.
I also fixed the Wi-Fi spelling error.

BrutusBondBTC commented:
Thanks for the feedback.


Sent from Proton Mail mobile



\-------- Original Message --------
On Feb 14, 2023, 5:30 PM, Michael Flaxman < ***@***.***> wrote:

>
>
>
> ***@***.***** requested changes on this pull request.
>
> Hotlinking is a bad practice. Please upload static assets to https://github.com/btcguide/btcguide.github.io/tree/master/assets/img
>
> --------------------
>
> In [\_pages/setup-wallets/paper-advanced.md][pages_setup-wallets_paper-advanced.md]:
>
> ```
> >  If you are truly paranoid, remove the hard drive from the machine (before booting up Tails) so there is no possibility that any private key material can be written to disk.
>
> +
> +**Watch A Video On How To Remove The Hard Drive and Wifi Card**
> +
> +[![AirGapComp](https://nostr.build/i/nostr.build_228a690dc06893b18976af8242284169329274d544ff0e1d22ea79015b338ee0.png)](https://youtu.be/AQuSs2LdARo)
> ```
>
> Is this anchor text intentional? It doesn't really make sense.
>
> --------------------
>
> In [\_pages/setup-wallets/paper-advanced.md][pages_setup-wallets_paper-advanced.md 1]:
>
> ```
> >  If you are truly paranoid, remove the hard drive from the machine (before booting up Tails) so there is no possibility that any private key material can be written to disk.
>
> +
> +**Watch A Video On How To Remove The Hard Drive and Wifi Card**
> ```
>
> This is spelled `Wi-Fi`
>
> —
> Reply to this email directly, [view it on GitHub][], or [unsubscribe][].
> You are receiving this because you authored the thread.![AYXVLOQPGUXSRHS5FXGP34LWXQWSVA5CNFSM6AAAAAAU4HXV3SWGG33NNVSW45C7OR4XAZNRKB2WY3CSMVYXKZLTORJGK5TJMV32UY3PNVWWK3TUL5UWJTSNNAL7Q.gif][]Message ID: ***@***.***>


[pages_setup-wallets_paper-advanced.md]: https://github.com/btcguide/btcguide.github.io/pull/124#discussion_r1106554328
[pages_setup-wallets_paper-advanced.md 1]: https://github.com/btcguide/btcguide.github.io/pull/124#discussion_r1106554766
[view it on GitHub]: https://github.com/btcguide/btcguide.github.io/pull/124#pullrequestreview-1298667512
[unsubscribe]: https://github.com/notifications/unsubscribe-auth/AYXVLOSEHBM5F57BOY5R6VTWXQWSVANCNFSM6AAAAAAU4HXV3Q
[AYXVLOQPGUXSRHS5FXGP34LWXQWSVA5CNFSM6AAAAAAU4HXV3SWGG33NNVSW45C7OR4XAZNRKB2WY3CSMVYXKZLTORJGK5TJMV32UY3PNVWWK3TUL5UWJTSNNAL7Q.gif]: https://github.com/notifications/beacon/AYXVLOQPGUXSRHS5FXGP34LWXQWSVA5CNFSM6AAAAAAU4HXV3SWGG33NNVSW45C7OR4XAZNRKB2WY3CSMVYXKZLTORJGK5TJMV32UY3PNVWWK3TUL5UWJTSNNAL7Q.gif
